pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/EtterlatteBrevRequest.kt

```diff
@@ -39,6 +39,7 @@ enum class EtterlatteBrevKode {
     OMSTILLINGSSTOENAD_VEDLEGG_FORHAANDSVARSEL_UTFALL,
     TILBAKEKREVING_INNHOLD,
     TILBAKEKREVING_FERDIG,
+    TILBAKEKREVING_DOEDSBO,
```

Comment:
Kanskje denne burde hete `TILBAKEKREVING_INNHOLD_DOEDSBO` slik at vi følger en form for navnestandard